main.js

```diff
@@ -38,3 +38,4 @@ export {
     tokenizer,
 } from "./lib/parse.js";
 export { to_ascii } from "./lib/minify.js";
+export { default_options } from "./tools/node";
```

Comment:
This should be moved to the `// CLI` section, `// TESTS` exports should be removed from the main bundle at some point.

Reply:
Good point, thx

Will update PR soon